code/modules/hydroponics/grown.dm

```diff
@@ -17,9 +17,9 @@
 	resistance_flags = FLAMMABLE
 	origin_tech = "biotech=1"
 
-/obj/item/reagent_containers/food/snacks/grown/New(newloc, var/obj/item/seeds/new_seed = null)
-	tastes = list(name = 1) // apples taste of apple, silly.
-	..()
+/obj/item/reagent_containers/food/snacks/grown/Initialize(mapload, obj/item/seeds/new_seed)
+	. = ..()
+	tastes = list("[name]" = 1) // apples taste of apple, silly.

```

Comment:
Any particular reason we're using `"[name]"` rather than `name`?

Reply:
Because name = means list(name=1) instead of list(apple=1)